projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -1019,30 +1151,42 @@ static function admin() {
 	/**
 	 * Content width setting callback
 	 */
-	static function content_width_settings() {
-		$safecss_post = Jetpack_Custom_CSS::get_current_revision();
+	public static function content_width_settings() {
+		$safecss_post = self::get_current_revision();
 
 		$custom_content_width = get_post_meta( $safecss_post['ID'], 'content_width', true );
 
 		// If custom content width hasn't been overridden and the theme has a content_width value, use that as a default.
-		if ( $custom_content_width <= 0 && ! empty( $GLOBALS['content_width'] ) )
+		if ( $custom_content_width <= 0 && ! empty( $GLOBALS['content_width'] ) ) {
 			$custom_content_width = $GLOBALS['content_width'];
+		}
 
-		if ( ! $custom_content_width || ( isset( $GLOBALS['content_width'] ) && $custom_content_width == $GLOBALS['content_width'] ) )
+		if ( ! $custom_content_width || ( isset( $GLOBALS['content_width'] ) && $custom_content_width == $GLOBALS['content_width'] ) ) { // phpcs:ignore Universal.Operators.StrictComparisons.LooseEqual
 			$custom_content_width = '';
+		}
 
 		?>
 		<div class="misc-pub-section">
 			<label><?php esc_html_e( 'Media Width:', 'jetpack' ); ?></label>
-			<span id="content-width-display" data-default-text="<?php esc_attr_e( 'Default', 'jetpack' ); ?>" data-custom-text="<?php esc_attr_e( '%s px', 'jetpack' ); ?>"><?php echo $custom_content_width ? sprintf( esc_html__( '%s px', 'jetpack' ), $custom_content_width ) : esc_html_e( 'Default', 'jetpack' ); ?></span>
+			<span id="content-width-display" data-default-text="<?php esc_attr_e( 'Default', 'jetpack' ); ?>" data-custom-text="
+				<?php
+					// translators: the custom content width.
+					esc_attr_e( '%s px', 'jetpack' );
+				?>
+				">
+				<?php
+					// translators: the custom content width.
+					echo $custom_content_width ? sprintf( esc_html__( '%s px', 'jetpack' ), $custom_content_width ) : esc_html_e( 'Default', 'jetpack' ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+				?>
+			</span>
 			<a class="edit-content-width hide-if-no-js" href="#content-width"><?php echo esc_html_e( 'Edit', 'jetpack' ); ?></a>
 			<div id="content-width-select" class="hide-if-js">
 				<input type="hidden" name="custom_content_width" id="custom_content_width" value="<?php echo esc_attr( $custom_content_width ); ?>" />
 				<p>
 					<?php
 
 					printf( /* translators: %1$s is replaced with an input field for numbers. */
-						__( 'Limit width to %1$s pixels for full size images. (<a href="%2$s" rel="noopener noreferrer" target="_blank">More info</a>.)', 'jetpack' ),
+						__( 'Limit width to %1$s pixels for full size images. (<a href="%2$s" rel="noopener noreferrer" target="_blank">More info</a>.)', 'jetpack' ), // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
This should do here.
```suggestion
						wp_kses_post( __( 'Limit width to %1$s pixels for full size images. (<a href="%2$s" rel="noopener noreferrer" target="_blank">More info</a>.)', 'jetpack' ) ),
```
Or you could be more strict with something like `wp_kses( ..., array( 'a' => array( 'href' => true, 'rel' => true, 'target' => true ) ) )`, but I don't know that it's worth spelling that all out.